src/main/java/com/minecolonies/coremod/client/render/worldevent/highlightmanager/IHighlightRenderData.java

```diff
@@ -0,0 +1,35 @@
+package com.minecolonies.coremod.client.render.worldevent.highlightmanager;
+
+import com.minecolonies.coremod.client.render.worldevent.WorldEventContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.time.Duration;
+
+/**
+ * Interface for highlightable items.
+ */
+public interface IHighlightRenderData
```

Comment:
mby make start/stop defaulting to noop, mby add is `isInfinite { getDuration() == null }`